src/core/ddsi/src/q_radmin.c

```diff
@@ -303,9 +303,15 @@ static void nn_rbuf_release (struct nn_rbuf *rbuf);
 #define RMSGTRACE(...)              TRACE_CFG (rmsg, rmsg->chunk.rbuf->rbufpool->logcfg, __VA_ARGS__)
 #define RDATATRACE(rdata, ...)      TRACE_CFG ((rdata)->rmsg, (rdata)->rmsg->chunk.rbuf->rbufpool->logcfg, __VA_ARGS__)
 
-static uint32_t align8uint32 (uint32_t x)
+static uint32_t align_rmsg (uint32_t x)
 {
-  return (x + 7u) & (uint32_t)-8;
+  /* Align rmsg chunks to the larger of sizeof(void*) or 8.
+  Ideally, we would use C11's alignof(void *), but sizeof(void *) is equivalent
```

Comment:
`alignof` can be used on aggregate types, so ideally we’d use `alignof(struct nn_rmsg)` rather than assuming “the strictest alignment need of rmsg comes from either uint32 or void*”

Reply:
That makes sense.  I'll revise.